### PR TITLE
[SHIRO-802] Pr template

### PR DESCRIPTION
**Pull request by Palak-137**
According to the issue on Jira the pull_request_template.md needs to be updated.
So I added the guideline to add -Purn-its to activate profile test.

### CODE REVIEW

**Review by fpapon — CHANGES_REQUESTED** [review]
@Palak-137 please provide a PR with only the template changes commit

**Comment by Palak-137** [comment]
Yes i have revert the rest of changes.

**Review by fpapon — COMMENTED** [review]
the best way is to drop the commits with `git rebase -i HEAD~8`

File: lang/src/main/java/org/apache/shiro/lang/io/ClassResolvingObjectInputStream.java [hunk 1]
```diff
@@ -27,7 +27,8 @@
 import java.io.ObjectStreamClass;
 
 /**
- * Enables correct ClassLoader lookup in various environments (e.g. JEE Servers, etc).
```
Comment (bmarwell) [inline lang/src/main/java/org/apache/shiro/lang/io/ClassResolvingObjectInputStream.java:30]: This file must not be part of the commit. Please rebase and create the PR from a branch.

**Review by bmarwell — CHANGES_REQUESTED** [review]
Prepend `[SHIRO-802]` for the PR and for each commit which has todo with this issue (after rebasing, please edit your commit)

**Review by fpapon — APPROVED** [review]
LGTM
@Palak-137 thanks!

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -9,6 +9,7 @@ Following this checklist to help us incorporate your contribution quickly and ea
        is to use the JIRA issue title in the pull request title and in the first line of the commit message.
  - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
  - [ ] Run `mvn clean install apache-rat:check` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
+ - [ ] By default the project packages the tests in an artifact. To activate them Run `mvn clean test -Purn-its`.
```
Comment (bmarwell) [inline .github/pull_request_template.md:12]: typo: `run-its`, not `urn-its`.

**Review by bmarwell — CHANGES_REQUESTED** [review]
typo

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -9,6 +9,7 @@ Following this checklist to help us incorporate your contribution quickly and ea
        is to use the JIRA issue title in the pull request title and in the first line of the commit message.
  - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
  - [ ] Run `mvn clean install apache-rat:check` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
+ - [ ] By default the project packages the tests in an artifact. To activate them,activate the `run-its` profile:  `mvn clean test -Purn-its`.
```
Comment (bmarwell) [inline .github/pull_request_template.md:12]: 1. Still "urn-its".
2. test will not run its. You will need "clean verify".
3. nit: Please add a space after a comma (between "them,activate").

**Review by bmarwell — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 3 above)
Comment (bmarwell) [reply .github/pull_request_template.md:12]: Proposed: "to activate them, run `mvn clean test -Prun-its`.

**Comment by fpapon** [comment]
@Palak-137 can you squash the commits please?

**Pull request closed without merging**